### DISCUSSION TITLE
Win 11 Runs Slow

### DISCUSSION THREAD

**Discussion by MCES10** [utmapp/UTM#5452] [General]
I am using a M1 Pro Macbook Pro and Windows 11 is really slow. I gave it 12Gb of Ram and lots of disk space. It has 1GHZ clock speed how do you increase that since parallels uses 2.5GHZ and runs faster .

**Comment by jsonmona** [comment]
That "1Ghz" is just an arbitrary number and does NOT affect performance. Are you sure parallels runs faster because of CPU? Parallels has GPU acceleration and UTM does not, and that might be the cause.

**Comment by MCES10** [comment]
Well if I enable UTM’s excelleration it doesn’t do anything to the peformance.

**Reply by jsonmona** [reply]
Of course. UTM's GPU acceleration only works with Linux guest. Since you're running Windows, it's almost meaningless.

*The word 'almost' is used because there is a small benefit when using -gl adapter with Windows. But it's quite small and won't make it run 3D games.*

**Comment by jsonmona** [comment]
Just to be clear, you've selected "Virtualize" and installed Windows 11 ARM64, right?

**Reply by MCES10** [reply]
Yes  I am running Windows 11 ARM
